src/viper/viper/api/backend/SilverBackend.scala

```diff
@@ -96,6 +96,8 @@ trait SilverBackend extends Backend with LazyLogging {
             }
         }
     }
+    // Early out of verification
+    if(skipVerification) return true
```

Comment:
I think this will potentially cache the wrong result, no? I think the stage should just make sure not to call Viper instead.

Reply:
Hmm Fair I'm not sure how the caching works, so I did not take that in account.

The reasoning of putting it here, is such that you still get output with the option `--backend-file-base`, but do not start verifying. This is (Which I found useful to debug/inspect the viper file).

Reply:
That's fair, let's split that into two stages then :) (so Col -> Silver, Silver -[verification]-> errors). I understand if you don't want to bother with that right now, but I don't want versions around that put wrong records in the cache, so maybe you can accept no viper file in the meantime.

Reply:
Yeah that seems fair, I'll have a look if this is possible in reasonable time.

I'll move some code from the 'submit' method.

Reply:
So I split the 'submit' method into a 'transform'  and 'submit' method. Adding an type for the intermediate program after the transform method was a bit of trouble, but this should work.